talkative-tangs/Bangazon-API
# Lb json db

## Link to Ticket
Todo item: create json for database loading

## Description of Proposed Changes
-Added a db.json file for loading into the database

## Steps to Test

```shell
git fetch --all
git checkout lb-json-db
```
To get a clean database you should delete your sqlite3 file and the migrations folder.  Then run:

```shell
python manage.py migrate
python manage.py makemigrations api
python manage.py migrate
python manage.py loaddata db.json
```

## Impacted Areas in Application
List general components of the application that this PR will affect:

* Models (I added a space on string method for readability)
*db.json
*db.sqlite3
*/migrations

## Mentions @username
@robbyhecht
@BryanNilsen
@ousamasama
